.github/actions/integration-tests/run-ci-stage2

```diff
@@ -2,6 +2,52 @@
 # -*- mode: sh; indent-tabs-mode: nil; sh-basic-offset: 4 -*-
 # vim: autoindent tabstop=4 shiftwidth=4 expandtab softtabstop=4 filetype=bash
 
+SCRIPT_DIR=$(dirname $0)
+SCRIPT_DIR=$(readlink -e ${SCRIPT_DIR})
+
+. ${SCRIPT_DIR}/base
+
+CI_RUN_ENVIRONMENT="standalone"
+
+longopts="run-environment:"
+opts=$(getopt -q -o "" --longoptions "${longopts}" -n "$0" -- "$@")
+if [ ${?} -ne 0 ]; then
+    echo "ERROR: Unrecognized option specified: $@"
+    exit 1
+fi
+eval set -- "${opts}"
+while true; do
+    case "${1}" in
+        --run-environment)
+            shift
+            CI_RUN_ENVIRONMENT="${1}"
+            shift
+            ;;
+        --)
+            shift
+            break
+            ;;
+        *)
+            echo "ERROR: Unexpected argument [${1}]"
+            shift
+            break
+            ;;
+    esac
+done
+
```

Comment:
can you just source it ? this is dup ?

Reply:
I thought about that, but some of the scripts have different options and I didn't want to make it too messy to add differing options to the ones that are the same at the moment.  I'm flexible though, do you think I should dedup it where possible?

Reply:
hmmm go ahead and merge it, we can consider this a small improvement, not a big deal